netket/operator/_sumoperators.py

```diff
@@ -85,7 +98,7 @@ def _expect_kernel_batched(
         return sum(result)
 
     def _pack_arguments(self):
-        return [self._coeff * jnp.array(op._pack_arguments()) for op in self._ops]
+        return list(self._coeff)
```

Comment:
```suggestion
        return self._coeff, [op._pack_arguments() for op in self._ops]
```

and then change expect_kernel_batched to
```python
    def _expect_kernel_batched(
        self, logpsi: Callable, params: PyTree, x: Array, data: Optional[PyTree]
    ):
    term_coefficients, term_datas = data

        result = [
            term_coefficients[i] * op._expect_kernel_batched(logpsi, params, x, term_datas[i])
            for i, op in enumerate(self._ops)
        ]
```